apps/gro/fields/address-outside.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = {
+  'address-text-one': {
+    validate: ['required'],
+  }
+};
```

Comment:
💩. I hadn't noticed. Yeah, @gxxm, keys of an object need to be different or one of them will be trounced.